src/Util/YamlSourceManipulator.php

```diff
@@ -216,6 +216,14 @@ private function updateData(array $newData)
                 continue;
             }
 
+            if (
+                $this->isMultilineString($newVal)
+                && rtrim($currentVal, "\n") !== $currentVal
+                && rtrim($newVal, "\n") === $newVal
+            ) {
+                $newVal .= "\n";
```

Comment:
Could you add a `$this->log()` line here that explains what you're doing? There's a little-known feature where, in `YamlSourceManipulatorTest.php`, you can uncomment a `setLogger()` line to see all these logs entries. And it helps document what each part does, since it's SO complex. For instance, in this case, I'm not entirely sure what edge-case this is fixing.

Reply:
Well, it seems not required. I think I went crazy and wanted to add/remove `\n` everywhere 😄.
I will remove completely